src/encoded/static/browser.js

```diff
@@ -2,11 +2,25 @@
 // Entry point for browser
 require('./libs/react-patches');
 var React = require('react');
+var ReactDOM = require('react-dom')
 var ReactMount = require('react/lib/ReactMount');
 ReactMount.allowFullPageRender = true;
 
 var App = require('./components');
 var domready = require('domready');
+var store = require('./store');
+import { Provider, connect } from 'react-redux';
```

Comment:
we have require and import, which after babel gets finished with them both end up as the same code (which is actually require) anyways I think we should standardise on using one for consistencies sake.